tests/templates/patterns/atoms/tags_test_atom/tags_test_atom.yaml

```diff
@@ -6,3 +6,10 @@ tags:
       raw: null
     zero:
       raw: 0
+  default_tag:
+    page.url:
+      raw: "example"
+    page.child.url:
+      raw: "another_example"
+    # None supplied by default value.
```

Comment:
This comment doesn't really make sense. Should it be "None _handled_ by default value"?

Reply:
I see your point, I've update to the above.